src/schema.rs

```diff
@@ -125,13 +126,26 @@ impl TryCmd {
         id: Option<&str>,
         stdout: Option<&crate::Data>,
         stderr: Option<&crate::Data>,
+        exit: Option<ExitStatus>,
     ) -> Result<(), crate::Error> {
         if let Some(ext) = path.extension() {
             if ext == std::ffi::OsStr::new("toml") {
```

Comment:
We should have tests for both toml and trycmd files

Reply:
Do you know how to create `ExitStatus` reliably? I tried [ExitStatusExt::from_raw](https://doc.rust-lang.org/std/os/unix/process/trait.ExitStatusExt.html#tymethod.from_raw) but it doesn't work as expected since this is for waited statuses

Reply:
I was able to construct `ExitStatus` in hacky way and added some unit tests for the status overwriting features